ecosystem/lava-sdk/src/stateTracker/updaters/pairing_updater.ts

```diff
@@ -42,7 +42,7 @@ export class PairingUpdater {
   }
 
   // update updates pairing list on every consumer session manager
-  public async update() {
+  public async update(virtualEpoch: number) {
```

Comment:
pairing updater shouldn't know or care about virtual epochs, it should not update consumerSessionsManager, consumerSessionsManager should only care about virtual epochs in getSessions